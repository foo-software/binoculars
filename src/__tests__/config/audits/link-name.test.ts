import auditLinkName from '../../../config/audits/link-name';

describe('link-name audit', () => {
  it('should match snapshot', () => {
    expect(auditLinkName('en')).toMatchSnapshot();
  });
});
