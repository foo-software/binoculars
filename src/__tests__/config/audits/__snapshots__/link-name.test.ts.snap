// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`link-name audit should match snapshot 1`] = `
Object {
  "description": "Link text (and alternate text for images, when used as links) that is discernible, unique, and focusable improves the navigation experience for screen reader users. [Learn more](https://web.dev/link-name/).",
  "title": "Links have a discernible name",
}
`;
